jaeger-core/src/main/java/io/jaegertracing/reporters/RemoteReporter.java

```diff
@@ -46,17 +46,6 @@
   private final int closeEnqueueTimeout;
   private final Metrics metrics;
 
-  /**
-   * RemoteReporter takes a Sender object, and sends spans for a specific protocol, and transport.
-   * At this point in time the protocol (thrift) is tightly-coupled to  Reporter, but in the future it may not be.
-   *
-   * @deprecated use {@link RemoteReporter.Builder} with fluent API
-   **/
-  @Deprecated
-  public RemoteReporter(Sender sender, int flushInterval, int maxQueueSize, Metrics metrics) {
-    this(sender, flushInterval, maxQueueSize, DEFAULT_CLOSE_ENQUEUE_TIMEOUT_MILLIS, metrics);
-  }
-
   RemoteReporter(Sender sender, int flushInterval, int maxQueueSize, int closeEnqueueTimeout,
```

Comment:
should we remove this ctor as well? Having public ctors is what leads to proliferation of deprecated methods in the first place, it's better to use builder pattern for most core components. Builder can be overkill for smaller classes, but RemoteReporter is a critical component that can continue evolving.

E.g. I see that we already use builder for RemoteSampler.

Reply:
It is package private, so not exposed to the world.

I can make it private in this PR.